install.sh

```diff
@@ -26,7 +26,7 @@ case "$architecture" in
 esac
 
 #This will allow things to work without sudo
-sudo chown -R `id -u`:`id -g` /usr/local
+sudo chown -R `id -u`:`id -g` "${CREW_PREFIX}"
```

Comment:
Should be `"#{CREW_PREFIX}"`.

Reply:
it's a shell script.  not a ruby script.

Reply:
Oops, missed that.